docs/source/config_files.rst

```diff
@@ -30,6 +55,14 @@ You can also install files from pip in your ``environment.yml`` as well.
 Our example `environment.yml <https://github.com/binder-examples/python-conda_pip/blob/master/environment.yml>`_
 shows how one can specify a conda environment for repo2docker.
 
+You can also specify which Python version you are using with
```

Comment:
``environment.yml`` is the standard configuration file used by Anaconda, conda, and miniconda. Our example...

You can also...

*We will need to add Python 3.7 as soon as Travis supports as 3.7 has been released.